ui/apps/platform/src/Containers/Dashboard/PatternFly/SummaryCounts.tsx

```diff
@@ -80,24 +95,36 @@ function SummaryCounts() {
     };
 
     return (
-        <Split className="pf-u-flex-wrap">
-            {tileEntityTypes.map((tileEntity) => (
-                <Button
-                    key={tileEntity}
-                    variant={ButtonVariant.link}
-                    component={LinkShim}
-                    href={tileLinks[tileEntity]}
-                >
-                    <Stack className="pf-u-px-xs pf-u-px-sm-on-xl pf-u-align-items-center">
-                        <span className="pf-u-font-size-lg-on-md pf-u-font-size-sm pf-u-font-weight-bold">
-                            {tileData[tileEntity]}
-                        </span>
-                        <span className="pf-u-font-size-md-on-md pf-u-font-size-xs">
-                            {pluralize(tileEntity, tileData[tileEntity])}
-                        </span>
-                    </Stack>
-                </Button>
-            ))}
+        <Split className="pf-u-align-items-center">
+            <SplitItem isFilled>
+                <Split className="pf-u-flex-wrap">
+                    {tileEntityTypes.map((tileEntity) => (
+                        <Button
+                            key={tileEntity}
+                            variant={ButtonVariant.link}
+                            component={LinkShim}
+                            href={tileLinks[tileEntity]}
+                        >
+                            <Stack className="pf-u-px-xs pf-u-px-sm-on-xl pf-u-align-items-center">
+                                <span className="pf-u-font-size-lg-on-md pf-u-font-size-sm pf-u-font-weight-bold">
+                                    {tileData[tileEntity]}
+                                </span>
+                                <span className="pf-u-font-size-md-on-md pf-u-font-size-xs">
+                                    {pluralize(tileEntity, tileData[tileEntity])}
+                                </span>
+                            </Stack>
+                        </Button>
+                    ))}
+                </Split>
+            </SplitItem>
+            <div
+                style={{ fontStyle: 'italic' }}
+                className="pf-u-color-200 pf-u-font-size-sm pf-u-mr-md pf-u-mr-lg-on-lg"
+            >
+                {`Last updated ${dateFormatter.format(lastUpdate)} at ${timeFormatter.format(
+                    lastUpdate
```

Comment:
For your info, here is a similar example with current dateUtils:

https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Clusters/Components/SensorStatus.tsx#L54

How high a priority was this specific date format? which differs from everywhere else in UI, as far as I know.

@vjwilson do you have any thought about pro and con of diverging from reusable functions?

Reply:
Here is another precedent where the format coulda shoulda been extracted to dateUtils:

https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Network/Graph/Overlays/NodesUpdateSection.tsx#L46-L49

![Last_Updated_Network_Graph](https://user-images.githubusercontent.com/11862657/175398240-b52c196f-5a6b-4278-b05e-522389cd2900.png)

Reply:
Good insight, I'll bring this up with UX as well. Gut feeling is we should use the same date/time format for Clusters and this dashboard, and it makes sense to use just the time for network graph. Even if we leave the date off though I don't see a compelling reason for the respective time formats to be different.

Reply:
Late to the discussion, but just a  couple of thoughts
1. Eventually, we need to converge on what the rest of PatternFly/OpenShift uses, and if they don't have a standard, we need to tell them that our format is the standard and they should converge on it. ;) 
2. Even after that, we should code date/times in a way that they are internationalized, based on your local settings.

Reply:
@vjwilson PatternFly element for date and time that we have heard about in Design Share is still future.

@dvail Can you include in the ticket the issue of when (if ever) local time zone (as opposed to locale) which UI displays now is appropriate instead of universal time zone:
* which supports clear communication by global teams
* which usually correlates with log files

Reply:
If universal time is most appropriate for security, then the geekier format is tends to communicate it clearly.

Reply:
https://issues.redhat.com/browse/ROX-11527